### DISCUSSION TITLE
Terminal wants password, but won't let me enter it when downloading homebrew

### DISCUSSION THREAD

**Discussion by nancyEHC** [Homebrew/discussions#1525] [Getting started]
looks like this (see below) with a key icon that does nothing when clicked, and then with no way to get cursor into place to type into field:
==> Checking for `sudo` access (which may request your password).
Password:

(ETA: I've tried in both Safari and Chrome

**Comment by SMillerDev** [comment]
This is the way sudo works. You just have to type and press enter.

**Comment by nancyEHC** [comment]
Right but it doesn’t let me type anything. It’s as if there is no text field. 

Nancy Cincotta
516-316-5783

**Comment by SMillerDev** [comment]
You won't see the characters you type. But it does let you input them.

**Comment by nancyEHC** [comment]
Gotcha. Thanks 

Nancy Cincotta
516-316-5783
